DEVELOPING.rst

```diff
@@ -5,120 +5,383 @@
+* `pymdownx.superfences <https://facelessuser.github.io/pymdown-extensions/extensions/superfences/>`_
+* `mdx_math <https://pypi.org/project/python-markdown-math/>`_
+
+Working with document: LaTeX
+============================
+
+LaTeX is enabled with the `mdx_math` extension. Inline equations are between
+single ``$``, e.g. ``$E = m c^2$``. For standalone math, use ``\[...\]``.
+
+Working with documents: images
+==============================
+
+Sometimes the document pages may have illustrating images. The images should be
+placed into the `data/images` directory following the document slug. They can
+then be referred to in your Markdown content by means of links. Please check an
+existing documentation page such as
+``totem-releases-first-set-of-open-data.md`` and where it stores and how it
+loads the illustrating image ``totem-roman-pots-in-the-lhc-tunnel.jpeg``.
+
+After you add an image and reference it in your Markdown source file, you
+should load the image into the system:
+portal instance,
 
 .. code-block:: console
 
-   $ my_docker exec -it web cernopendata fixtures records --file /content/data/records/<full_path>
-   $ my_docker exec -it web cernopendata fixtures docs --file /content/data/docs/<full_path>
+   $ docker compose exec -it web /content/scripts/load-images.sh
+
+You should now be able to see the image locally in the document record.
+
+Appendix: Git workflow
+======================
+
+Here is detailed example of our `GitHub flow
+<https://guides.github.com/introduction/flow/index.html>`_.
 
-..
+Setting up repository
+---------------------
 
-5. Finally, if there are new images, ensure that they appear in the correct folder
+Let's assume your GitHub account name is ``johndoe``.
+
+Firstly, fork `opendata.cern.ch repository
+<https://github.com/cernopendata/opendata.cern.ch/>`_ by using the "Fork"
+button on the top right.  This will give you your personal repository:
 
 .. code-block:: console
 
-   $ my_docker exec -it web /content/scripts/load-images.sh
+   http://github.com/johndoe/opendata.cern.ch
+
+Secondly, clone this repository onto your laptop and set up remotes so that
+``origin`` would point to your repository and ``upstream`` would point to the
+canonical location:
 
-..
+.. code-block:: console
 
+   $ cd ~/private/src
+   $ git clone git@github.com:johndoe/opendata.cern.ch
+   $ cd opendata.cern.ch
+   $ git remote add upstream git@github.com:cernopendata/opendata.cern.ch
+
+Optionally, if you are also going to integrate work of others, you may want to
+set up `special PR branches
+<http://simko.home.cern.ch/simko/github-local-handling-of-pull-requests.html>`_
+like this:
+
+.. code-block:: console
+
+   $ vim .git/config
+   $ cat .git/config
+   [remote "upstream"]
+       url = git@github.com:cernopendata/opendata.cern.ch
+       fetch = +refs/heads/*:refs/remotes/upstream/*
+       fetch = +refs/pull/*/head:refs/remotes/upstream/pr/*
 
 Understanding repository branches
```

Comment:
Before the section of the repository branches, it would be nice to have a section with the repository structure, explaining the difference between the different folders (in particular the one of the `skeletons`).

Reply:
Added Appendix A explaining the repository structure, and also added Appendix C explaining the git conventional commit style (that is now being checked by the CI).